lib/cli-config.js

```diff
@@ -70,12 +72,16 @@ exports.getContent = function(config, directory) {
 
         if (ext === '.js' || ext === '.json') {
             content = require(configPath);
+        } else if (ext === '.yaml') {
+            content = yaml.safeLoad(fs.readFileSync(configPath, 'utf8'));
         } else {
-            content = JSON.parse(
-                stripJSONComments(
-                    fs.readFileSync(configPath, 'utf8')
-                )
-            );
+            data = fs.readFileSync(configPath, 'utf8');
+
+            try {
+                content = JSON.parse(stripJSONComments(data));
+            } catch (e) {
+                content = yaml.safeLoad(data);
```

Comment:
I don't really see why not to be honest... It's valid YAML.
Are you sure you wanna go down the road of what is pretty yaml vs. ugly yaml and do detection based on those assumptions?

In any case, if the YAML parsing fails (because they really don't know what they're doing), they *will* get a pretty JSON parse error with the code I wrote above.

Would you rather have it reversed? Try JSON first, if that fails, try YAML, if that fails throw an error? I can switch it around...